src/content/blog/astro-i18n.mdx

```diff
@@ -0,0 +1,110 @@
+- needing an update to match the English documentation;
+- or, not yet translated.
+
+<BlogContentImage src={trackerTable} alt="" />
+
+Internally, it uses the git history to compare changes between languages, defines the status of each page through some checks, and also generates a GitHub link with all the commits a page needs to be in sync with the English version. It’s powered by GitHub Actions, right out of our docs repo.
+
+<BlogContentImage src={trackerSummary} alt="" />
+
+Our Translation Tracker also uses GitHub data to list open translation PRs in need of reviews from native speakers, which are a great way to get started without the pressure of opening your own PR!
+
+<BlogContentImage src={translationReviews} alt="" />
+
+**With a clear picture of what our translation needs are, we are able to quickly welcome and onboard new i18n contributors who already know which contributions are wanted.**
+
+## Driving decisions through data
+
+Initially, we made translation decisions (e.g. which languages to support in docs) based solely on our community’s excitement and availability to help us. This proved to be a good strategy when we were just getting started. But as Astro docs grew, we needed to make more informed decisions.
+
+One of the difficult calls we had to make early on was to decide which languages we would officially support with translations. Or rather, which ones we would not. Officially supporting a language is quite a large committment, which unfortunately means that we cannot accept every language proposed to us. Our goal was to find a balance between languages easier for us to maintain and the ones that would make the most impact. Access to quality information and education isn’t the same everywhere — and as the ones with the privilege of English proficiency ourselves, we are in a position of strength to address these inequities and advocate for the communities who need translations the most.
+
+To help us make these difficult decisions, we analyzed several data points such as the number of members for each language in our Discord, a list of countries per language, demographics of several developer surveys, and [EF’s English Proficiency Index](https://www.ef.com/wwpt/epi/). By starting with clear goals, we could choose languages that helped keep us focused on meeting the needs of the global developer community.
+
+## Supporting our contributors
+
+The last couple of months in particular have been dedicated to supporting and recognizing the massive work done by our internationalization teams. While Astro Docs has long had a “Face Pile” of the GitHub avatars of every contributor to the repository, we now display a [dedicated translators’ face pile](https://docs.astro.build/es/getting-started/#traducir) on each translated version of the site. Our popular community [Astro Badges](https://astro.badg.es/v1/achievements/) also now display dedicated contributions to acknowledge both translations and translation reviews.
```

Comment:
We have "dedicated" three times here, maybe we can diversify this a bit? I was thinking something like this, but feel free to run with whatever makes more sense for you!
```suggestion
The last couple of months in particular have been focused on supporting and recognizing the massive work done by our internationalization teams. While Astro Docs has long had a “Face Pile” of the GitHub avatars of every contributor to the repository, we now display a [dedicated translators’ face pile](https://docs.astro.build/es/getting-started/#traducir) on each translated version of the site. Our popular community [Astro Badges](https://astro.badg.es/v1/achievements/) also now displays unique i18n contributions to acknowledge both translations and translation reviews.
```